sql/moz-fx-data-shared-prod/static/normal_distribution/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+friendly_name: Normal distribution
+workgroup_access:
+  - role: roles/bigquery.dataViewer
+    members:
+      - workgroup:mozilla-confidential
+      - workgroup:pocket/external
```

Comment:
Is metadata and `workgroup_access` supported for static datasets? Or should I add this in the cloudops-infra repo instead? https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-shared/tf/prod/envs/prod/bigquery-new/namespaces.auto.tfvars.json

Reply:
`workgroup_access` is supported for static datasets.